radar-desktop fails to run on Fedora 43 KDE

## Describe the bug

The radar-desktop app (v1.4.4) doesn't run on Fedora 43 (KDE Edition) running Wayland.

## To reproduce

Steps to reproduce the behavior:

1. Launch radar-desktop after installing the RPM
2. There is a brief window (black) that appears and then disappears on screen
3. The output on the console is:
```
2026/04/14 22:10:17 Radar Desktop 1.4.4 starting...
2026/04/14 22:10:18 PATH enriched from login shell (6 entries)
2026/04/14 22:10:18 KUBECONFIG enrichment skipped: not found in login shell
2026/04/14 22:10:18 [theme] Could not read color-scheme from xdg-desktop-portal: No such interface “org.freedesktop.appearance”
2026/04/14 22:10:18 Kubeconfig loaded: mode=single, files=1, contexts=1, exec-plugins=1
2026/04/14 22:10:18 Using kubeconfig: /home/jonny/.kube/config
2026/04/14 22:10:18 MCP server enabled (port will be assigned at startup)
2026/04/14 22:10:18 Cleaned image layer cache directory
2026/04/14 22:10:18 Image layer cache initialized at: /tmp/radar-image-cache
2026/04/14 22:10:18 SSE broadcaster: cache not ready, waiting for connection...
2026/04/14 22:10:18 Starting Explorer server on http://localhost:41987
2026/04/14 22:10:18 [mcp] Port file written: /home/jonny/.radar/mcp-port (port 41987)
2026/04/14 22:10:18 [ops] Canceling ongoing operations (previous API calls will be interrupted)
2026/04/14 22:10:18 [ops] InitializeCluster START (context=teleport.ervine.cloud-homeK8s)
2026/04/14 22:10:18 SSE broadcaster: connection state changed to "connecting" (context=teleport.ervine.cloud-homeK8s, progress="Testing cluster connectivity...")
2026/04/14 22:10:18 Initialized in-memory event store (max 10000 events)
Overriding existing handler for signal 10. Set JSC_SIGNAL_FOR_GC if you want WebKit to use a different signal
2026/04/14 22:10:18 API resource discovery took 251.127383ms
2026/04/14 22:10:18 Discovered 272 API resources (259 unique kinds)
2026/04/14 22:10:18 SSE broadcaster: connection state changed to "connecting" (context=teleport.ervine.cloud-homeK8s, progress="Loading workloads...")
2026/04/14 22:10:18 Starting resource cache: 10 critical + 9 deferred + 1 background informers (20 total)
Gdk-Message: 22:10:19.021: Error 71 (Protocol error) dispatching to Wayland display.
```

## Expected behavior

The application should launch.

## Screenshots

## Diagnostics (optional)

Setting GDK_BACKEND=X11 and running radar-desktop also fails - details below:
```
DK_BACKEND=X11 radar-desktop 
2026/04/14 22:12:45 Radar Desktop 1.4.4 starting...
2026/04/14 22:12:46 PATH enriched from login shell (6 entries)
2026/04/14 22:12:46 KUBECONFIG enrichment skipped: not found in login shell
2026/04/14 22:12:46 [theme] Could not read color-scheme from xdg-desktop-portal: No such interface “org.freedesktop.appearance”
2026/04/14 22:12:46 Kubeconfig loaded: mode=single, files=1, contexts=1, exec-plugins=1
2026/04/14 22:12:46 Using kubeconfig: /home/jonny/.kube/config
2026/04/14 22:12:46 MCP server enabled (port will be assigned at startup)
2026/04/14 22:12:46 Cleaned image layer cache directory
2026/04/14 22:12:46 Image layer cache initialized at: /tmp/radar-image-cache
2026/04/14 22:12:46 SSE broadcaster: cache not ready, waiting for connection...
2026/04/14 22:12:46 Starting Explorer server on http://localhost:38653
2026/04/14 22:12:46 [mcp] Port file written: /home/jonny/.radar/mcp-port (port 38653)
2026/04/14 22:12:46 [ops] Canceling ongoing operations (previous API calls will be interrupted)
2026/04/14 22:12:46 [ops] InitializeCluster START (context=teleport.ervine.cloud-homeK8s)
2026/04/14 22:12:46 SSE broadcaster: connection state changed to "connecting" (context=teleport.ervine.cloud-homeK8s, progress="Testing cluster connectivity...")
2026/04/14 22:12:46 Initialized in-memory event store (max 10000 events)
panic: failed to init GTK

goroutine 1 [running, locked to thread]:
github.com/wailsapp/wails/v2/internal/frontend/desktop/linux.NewFrontend.func1()
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/internal/frontend/desktop/linux/frontend.go:160 +0x136
sync.(*Once).doSlow(0x14a7a34?, 0x7ff4f17175a0?)
        /opt/hostedtoolcache/go/1.26.2/x64/src/sync/once.go:78 +0xac
sync.(*Once).Do(...)
        /opt/hostedtoolcache/go/1.26.2/x64/src/sync/once.go:69
github.com/wailsapp/wails/v2/internal/frontend/desktop/linux.NewFrontend({0x5039750, 0x509463175f0}, 0x50946318000, 0x50946308090, 0x50945d01dc0, {0x501b840, 0x50945e9e960})
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/internal/frontend/desktop/linux/frontend.go:151 +0x77
github.com/wailsapp/wails/v2/internal/frontend/desktop.NewFrontend(...)
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/internal/frontend/desktop/desktop_linux.go:16
github.com/wailsapp/wails/v2/internal/app.CreateApp(0x50946318000)
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/internal/app/app_production.go:86 +0x5af
github.com/wailsapp/wails/v2/pkg/application.(*Application).Run(0x509463140e0)
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/pkg/application/application.go:58 +0x25
github.com/wailsapp/wails/v2.Run(0x50946318000?)
        /home/runner/go/pkg/mod/github.com/wailsapp/wails/v2@v2.12.0/wails.go:14 +0x9a
main.main()
        /home/runner/work/radar/radar/cmd/desktop/main.go:152 +0x1645
```

- Radar version: 1.4.4
- Kubernetes:1.34.3 (kubespray)

--- a/cmd/desktop/boot_env_linux.go
+++ b/cmd/desktop/boot_env_linux.go
@@ -25,6 +25,26 @@ func logBootEnv() {
 	}
 }
 
+// applyWebKitDefaults sets WEBKIT_DISABLE_DMABUF_RENDERER=1 unless the user
+// has set it explicitly. WebKitGTK's DMABUF renderer produces blank windows
+// on a range of Wayland setups (NVIDIA, KDE KWin, some Mesa stacks) and
+// upstream considers this an application-level concern, not a WebKit fix.
+// The legacy renderer is stable everywhere.
+//
+// Must run before Wails initializes WebKit. LookupEnv (not an empty-string
+// check) lets users opt back in to DMABUF with WEBKIT_DISABLE_DMABUF_RENDERER=0.
+func applyWebKitDefaults() {
+	const key = "WEBKIT_DISABLE_DMABUF_RENDERER"
+	if _, set := os.LookupEnv(key); set {
+		return
+	}
+	if err := os.Setenv(key, "1"); err != nil {
+		log.Printf("[desktop] failed to set %s: %v", key, err)
+		return
+	}
+	log.Printf("[desktop] applied %s=1 (default; set %s=0 to opt out)", key, key)
+}
+
 // joinEnv formats env vars as "KEY=value" pairs. When includeUnset is true,
 // unset vars are rendered as "KEY=" so the reader can tell they were checked.
 // When false, unset vars are omitted (noise reduction for overrides).

--- a/cmd/desktop/boot_env_other.go
+++ b/cmd/desktop/boot_env_other.go
@@ -3,3 +3,5 @@
 package main
 
 func logBootEnv() {}
+
+func applyWebKitDefaults() {}

--- a/cmd/desktop/main.go
+++ b/cmd/desktop/main.go
@@ -64,6 +64,11 @@ func main() {
 	// No-op on macOS/Windows.
 	logBootEnv()
 
+	// Disable WebKit's DMABUF renderer on Linux unless the user opts in —
+	// it produces blank windows on Wayland+KDE/NVIDIA and upstream won't fix.
+	// Must run before Wails initializes WebKit.
+	applyWebKitDefaults()
+
 	// GUI apps (macOS .app, Linux .desktop) get a minimal PATH that
 	// doesn't include user-installed tools like gke-gcloud-auth-plugin,
 	// gcloud, aws CLI, etc. Enrich PATH from the user's login shell.

